### DISCUSSION TITLE
Is it possible to enable kepub visibility within OPDS?

### DISCUSSION THREAD

**Discussion by mrecom** [koreader/koreader#9388] [Q&A]
My primary use for kepub is within the OPDS. I'd like to be able to download kepub files, but they do not show up within OPDS for some reason. Any help is appreciated.

Best regards

**Comment by pazos** [comment]
kepub is actually `kepub.epub` so it will be shown as an epub, which the opds client can handle KO ko can read.

**Reply by NiLuJe** [reply]
There's a bit of trickery involved in calibre in that if you actually *store* KePubs in your calibre library, they *do* use the `.kepub` extension :/.

It's only after a Send To Device that the switcheroo happens...
